src/main/resources/assets/minecolonies/book/minecolonies.xml

```diff
@@ -0,0 +1,1463 @@
+<book title="Minecolonies Resurrection" cover="gbook:covers/default" fontSize="0.6">
```

Comment:
BTW this was NOT done by me, i just updated the version we use so that it actually loads